Release Conversations Language Understanding SDK 1.1.0-beta.1

heaths opened the pull request:
* Convert to DPG with HLC models
* Ignore long sync LRO test

  Caused by #29140 (seemingly)

* Add swagger transforms

  Works around #29141 and #29143

* Convert to DPG

  Also fixes #26379

* Update public APIs and documentation
* Resolve PR feedback
* Resolve offline feedback
* Update generated code

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.AI.Language.Conversations](https://apiview.dev/Assemblies/Review/6b5eb26180814f04b275cf46894a34ab)

heaths committed:
Prepare Conversations Language Understanding SDK 1.1.0-beta.1

* Convert to DPG with HLC models
* Ignore long sync LRO test

  Caused by Azure#29140 (seemingly)

* Add swagger transforms

  Works around Azure#29141 and Azure#29143

* Convert to DPG

  Also fixes Azure#26379

* Update public APIs and documentation
* Resolve PR feedback
* Resolve offline feedback
* Update generated code

kristapratico commented on sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md:
```diff
@@ -0,0 +1,103 @@
+    },
+};
+
+Operation<BinaryData> analyzeConversationOperation = client.SubmitJob(WaitUntil.Started, RequestContent.Create(data));
```
I thought it was called AnalyzeConversation?

heaths replied:
It is. I forgot to run our snippet updater.

kristapratico commented on sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationSummarization.md:
````diff
@@ -0,0 +1,103 @@
+Using the same `data` definition above, you can make an asynchronous request by calling `AnalyzeConversationAsync`:
+
+```C# Snippet:AnalyzeConversationAsync_ConversationSummarization
+Operation<BinaryData> analyzeConversationOperation = await client.SubmitJobAsync(WaitUntil.Started, RequestContent.Create(data));
````
same here (and across all samples it looks like)

heaths replied:
Same resolution.

heaths committed:
Stop always recording authoring tests

heaths committed:
Update samples

heaths committed:
Update CHANGELOG for release
